addons/test_mail_full/tests/test_ui.py

```diff
@@ -69,6 +69,15 @@ def test_message_actions_without_login(self):
             "message_actions_tour",
         )
 
+    def test_message_actions_portal_when_follower(self):
+        portal_user = self._create_portal_user()
+        self.record_portal.message_subscribe(partner_ids=[portal_user.partner_id.id])
```

Comment:
I don't think being follower/calling message_subscribe is relevant if we don't add the `mail_test_portal_rule_portal` rule.

The problematic part is not "when follower" but more precisely "when ACL permits access to the thread".

This should just be replaced by a check at the start of the test that portal user indeed has access to the thread.